app/invocation/invocation_not_found.tsx

```diff
@@ -37,8 +28,36 @@ export default class InvocationNotFoundComponent extends React.Component {
               </div>
             </div>
           </div>
-        )}
-      </div>
-    );
+        </div>
+      );
+    }
+
+    if (this.props.isAuthenticated) {
```

Comment:
should need this if since the `!this.props.isAuthenticated` case returns

Reply:
Ah yep, good catch